src/state/configure-store.ts

```diff
@@ -4,14 +4,29 @@ import { createLogicMiddleware } from "redux-logic";
 
 import { BASE_API_URL, DATA_BUCKET_URL } from "../constants";
 
-import { enableBatching, initialState, metadata, selection, State } from "./";
+import {
+    enableBatching,
+    initialState,
+    trajectory,
+    selection,
+    viewer,
+    simularium,
+    State,
+} from "./";
 
 const reducers = {
-    metadata: metadata.reducer,
+    trajectory: trajectory.reducer,
     selection: selection.reducer,
+    viewer: viewer.reducer,
+    simularium: simularium.reducer,
 };
 
-const logics = [...metadata.logics, ...selection.logics];
+const logics = [
+    ...trajectory.logics,
+    ...selection.logics,
+    ...viewer.logics,
+    ...simularium.logics,
+];
```

Comment:
same as above comment.  do these lists need to always be complete?

Reply:
That's a good question. @meganrm what do you think? Right now `selection.logics` and `simularium.logics` are just empty arrays, but I thought I'd include them here now so we don't have to worry about it down the road if/when we do add logic functions to those branches.

Reply:
It's also good to know that if you add more state you have to come here and insert the new stuff.

Reply:
yeah, so @schoinh you're right that it's better to add the empty arrays. It can be really frustrating working on some logic that hits an external source and have it not work because you forgot it actually import it into state. 

@toloudis the separation of state into the different folders/branches is purely for our own organization and sanity. Redux only has one big reducer. Thats why we take the trouble to make sure the constants we use in the reducer are named for their branch, otherwise you can have namespace overlap and have an action get called you don't expect.